.github/workflows/main.yml

```diff
@@ -0,0 +1,14 @@
+# Display a link to the artifacts at the bottom of a PR
+# from https://github.com/marketplace/actions/run-circleci-artifacts-redirector?version=0.1.0
+on: [status]
+jobs:
+  circleci_artifacts_redirector_job:
+    runs-on: ubuntu-latest
+    name: Run CircleCI artifacts redirector
+    steps:
+      - name: GitHub Action step
+        uses: larsoner/circleci-artifacts-redirector-action@master
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          artifact-path: 0/doc/index.html
+          circleci-jobs: build_doc
```

Comment:
You probably need to use the job names not the step names, so `quick-build,full-build` not `build_doc`